syntax = "proto3";

package proto_info;

message RequestMsg {
    bytes client_id = 1;
    bytes server_id = 2;
    bytes sign = 3;
    bytes data = 4;
}

message RespondMsg {
	uint32 seckey_id = 1;
	bytes client_id = 2;
	bytes server_id = 3;
	bytes data = 4;
}
